AntennaTracker/Parameters.cpp

```diff
@@ -387,6 +387,25 @@ const AP_Param::Info Tracker::var_info[] = {
     // @Units: d%
     // @User: Advanced
 
+    // @Param: PITCH2SRV_D_FF
+    // @DisplayName: Pitch Derivative FeedForward Gain
+    // @Description: FF D Gain which produces an output that is proportional to the rate of change of the target
+    // @Range: 0.001 0.1
```

Comment:
range should start with 0, or default is out of range

Reply:
Fixed